src/base/net/downloadmanager.cpp

```diff
@@ -148,10 +148,20 @@ Net::DownloadManager::DownloadManager(QObject *parent)
         QStringList errorList;
         for (const QSslError &error : errors)
             errorList += error.errorString();
-        LogMsg(tr("Ignoring SSL error, URL: \"%1\", errors: \"%2\"").arg(reply->url().toString(), errorList.join(u". ")), Log::WARNING);
 
-        // Ignore all SSL errors
-        reply->ignoreSslErrors();
+        QString errorMsg;
+        if (!(Preferences::instance()->isIgnoreSSLErrors()))
```

Comment:
```suggestion
        if (!Preferences::instance()->isIgnoreSSLErrors())
```

Reply:
OK, I thought it added to the readability...